src/main/java/org/recordrobotics/charger/control/IControlInput.java

```diff
@@ -18,4 +18,11 @@ public interface IControlInput {
 	 * @return [-1, 0) - left; (0, 1] - right
 	 */
 	double getDriveLat();
+
+	/**
+	 * Claw state input (cone & cube) value
+	 *
+	 * @return 1 Cube - -1 Cone - 0 neutral
```

Comment:
This is hard to read, dash isn't a very good separator.
On another note this is a job better suited for enums

Reply:
done

Reply:
did you add an enum? @vladaviedov is an enum the right choice here?